src/ui/articleview.cc

```diff
@@ -724,14 +726,14 @@ bool ArticleView::eventFilter( QObject * obj, QEvent * ev )
 
     return handled;
   }
-#endif
 
   if ( ev->type() == QEvent::MouseMove ) {
     if ( Gestures::isFewTouchPointsPresented() ) {
       ev->accept();
       return true;
     }
   }
+#endif
```

Comment:
Caught a little problem here. Old code did not exclude this "Gestures::isFewTouchPointsPresented" which is useless on macOS.

Reply:
Is this fixed now?